WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -32,7 +32,37 @@ def new
   end
 
   def index
-    @competitions = Competition.all.select([:id, :name, :cityName, :countryId]).order(:year, :month, :day).reverse_order
+    query = "CAST(CONCAT(year,'-',month,'-',day) as Datetime) > ? and showAtAll = true"
+    query_params = [(Date.today - 90)]
+
+    @regions = [ ["All","all"],["",""],["Africa","_Africa"],["Asia","_Asia"],["Europe","_Europe"],["North America","_North America"],["Oceania","_Oceania"],["South America","_South America"],["",""] ] + Country.all.map { |country| [country.name, country.id] }
```

Comment:
Can we add a method to Country.rb to get a list of all continents? I don't like that we've copied them all here.

What are the empty strings for? Just spacing? Can we use [optgroup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) instead?